File: src/com/scala/Homework/Student.java
package com.scala.Homework;

public class Student {
    private String name;
    private String cname;
    private double grade;

    public Student(String name, String cname, double grade) {
        this.name = name;
        this.cname = cname;
        this.grade = grade;
    }

    @Override
    public String toString() {
        return "name='" + name + '\'' + ", cname='" + cname + '\'' + ", grade=" + grade;
    }
}

File: src/com/scala/Homework/Homework05/Student.scala
package com.scala.Homework.Homework05

class Student(override val age:Int,override val name:String,val grade:Int,val pro:String,val sn:String) extends Person {

  override def printMessage: Unit = {
    printf("age = %d, name = %s, grade = %d, profession = %s, schoolName = %s", super.getAge, super.getName, grade, pro, sn)
  }
}
